### ISSUE TITLE
Add ability to specify exceptions to rules

### ISSUE THREAD

**Issue by jregistr** [cjoudrey/graphql-schema-linter#187]
In short, I'm wondering if we'd be open to adding a way to specify lint ignore options per field/type.

As an example, say we had a rule
**Type-Name-Longer-Than-1-Char**

```gql
type Foobar {}
type A {} # Let's provide an exception for this type.
type B {} # But this one should cause an error
```

We could achieve this via config options. But then it would not be immediately obvious when looking at the schema which ones are given exceptions to the rule.

Has anyone else thought about this? Perhaps we can collect ideas here.

**Comment by cjoudrey** [comment]
Hey @jregistr! Thanks for opening this issue.

I believe this is quite similar to what was discussed here: https://github.com/cjoudrey/graphql-schema-linter/issues/18

**Comment by jregistr** [comment]
Ah, this has been asked before. I'm closing this issue then and watching that other discussion.
